pkg/cloud/cloud.go

```diff
@@ -1527,20 +1532,28 @@ func (c *cloud) listSnapshots(ctx context.Context, request *ec2.DescribeSnapshot
 }
 
 // waitForVolume waits for volume to be in the "available" state.
-// On a random AWS account (shared among several developers) it took 4s on average.
 func (c *cloud) waitForVolume(ctx context.Context, volumeID string) error {
+	// Based on our testing in us-west-2 and ap-south-1, the median/p99 time until volume creation is ~1.5/~4 seconds.
+	// We have found that the following parameters are optimal for minimizing provisioning time and DescribeVolumes calls
+	// we queue DescribeVolume calls after [1.25, 0.5, 0.75, 1.125, 1.7, 2.5, 3] seconds.
+	// In total, we wait for ~60 seconds.
 	var (
-		checkInterval = 3 * time.Second
-		// This timeout can be "ovewritten" if the value returned by ctx.Deadline()
-		// comes sooner. That value comes from the external provisioner controller.
-		checkTimeout = 1 * time.Minute
+		initialDelay = volumeCreationPollInitialDelay
+		backoff      = wait.Backoff{
+			Duration: volumeCreationPollBackoffDuration,
+			Factor:   volumeCreationPollBackoffFactor,
+			Steps:    volumeCreationPollBackoffSteps,
+			Cap:      volumeCreationPollBackoffCap,
+		}
 	)
 
+	time.Sleep(initialDelay)
```

Comment:
Can this just be `time.Sleep(volumeCreationPollInitialDelay)`? Why the extra variable?
Otherwise lgtm